### PR TITLE
Final fixes 66

### PR DESCRIPTION
**Pull request by madhavmansuriya40**

### CODE REVIEW

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 1]
```diff
@@ -126,6 +126,9 @@ $red: #f00;
     line-height: 1.5;
     white-space: nowrap;
     min-width: 120px;
+    max-width: 170px;
+    display: contents;
```
Comment (Julias0) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:130]: what is this?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 2]
```diff
@@ -153,8 +156,10 @@ $red: #f00;
     font-weight: 500;
     line-height: 2;
     white-space: nowrap;
-    display: flex;
     min-width: 120px;
+    max-width: 170px;
+    display: contents;
```
Comment (Julias0) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:161]: ?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss (hunk 1 above)
Comment (Julias0) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:130]: Please use flexbox only

File: src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.html [hunk 3]
```diff
@@ -322,6 +322,7 @@
                             [mandatory]="true"
                             [options]="subCategories"
                             [selectModalHeader]="'Choose Sub Category'"
+                            [nullOption]="false"
```
Comment (Julias0) [inline src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.html:325]: why?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss (hunk 1 above)
Comment (madhavmansuriya40) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.scss:130]: max width with display flex was not working

File: src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.html (hunk 3 above)
Comment (madhavmansuriya40) [reply src/app/fyle/add-edit-per-diem/add-edit-per-diem.page.html:325]: its not there in old app

File: src/app/fyle/add-edit-expense/add-edit-expense.page.scss [hunk 4]
```diff
@@ -125,7 +125,9 @@ $red: #f00;
     font-weight: 500;
     line-height: 1.5;
     white-space: nowrap;
-    min-width: 120px;
+    display: flex;
+    max-width: 170px;
```
Comment (Julias0) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.scss:129]: arrey add this with a modifier
```
&__large {
max-width: 170px;
}

```

File: src/app/fyle/my-add-edit-trip/other-requests/other-requests.component.scss [hunk 5]
```diff
@@ -80,7 +79,7 @@
     font-weight: 500;
     line-height: 1.5;
     height: 24px;
-    max-width: calc(100% - 120px);
+    max-width: calc(100% - 170px);
```
Comment (Julias0) [inline src/app/fyle/my-add-edit-trip/other-requests/other-requests.component.scss:82]: same as above

**Review by Julias0 — APPROVED** [review]

**Pull request closed without merging**